pom.xml

```diff
@@ -23,14 +23,19 @@
     <artifactId>hivemq-influxdb-extension</artifactId>
     <properties>
         <extension.name>InfluxDB Monitoring Extension</extension.name>
+        <extension.author>HiveMQ</extension.author>
+
+        <asciidoctor.version>1.5.2.1</asciidoctor.version>
+        <output.dir>${basedir}</output.dir>
+        <source.document.name>README.adoc</source.document.name>
     </properties>
-    <version>4.0.0</version>
+    <version>4.1.0</version>
 
     <dependencies>
         <dependency>
             <groupId>com.hivemq</groupId>
             <artifactId>hivemq-extension-sdk</artifactId>
-            <version>4.0.0</version>
+            <version>4.2.0</version>
```

Comment:
We should not update the extension SDK to prevent any usage of APIs that are not available in older versions.